libnavui-maps/api/current.txt

```diff
@@ -1050,6 +1050,7 @@ package com.mapbox.navigation.ui.maps.route.line.model {
     method public boolean getDisplayRestrictedRoadSections();
     method public boolean getDisplaySoftGradientForTraffic();
     method public com.mapbox.maps.extension.style.layers.properties.generated.IconPitchAlignment getIconPitchAlignment();
+    method public double getLineDepthOcclusionFactor();
```

Comment:
Should it have `ExperimentalPreviewMapboxNavigationAPI` annotation as well?

Reply:
It should, but it's metalava behaving strangely. I ran `make ui-update-api`.